prow/tide/tide.go

```diff
@@ -803,7 +803,7 @@ func unsuccessfulContexts(contexts []Context, cc contextChecker, log *logrus.Ent
 		"total_context_count":  len(contexts),
 		"context_names":        contextsToStrings(contexts),
 		"failed_context_count": len(failed),
-		"failed_context_names": contextsToStrings(contexts),
+		"failed_context_names": contextsToStrings(failed),
```

Comment:
https://maxchadwick.xyz/blog/testing-log-output-in-go-logrus, a unit test would be appreciated

Reply:
Added supporting UT for the changes.

Reply:
cool